lib/rspec/rails/matchers/active_job.rb

```diff
@@ -169,6 +169,10 @@ def set_expected_number(relativity, count)
                                end
           end
 
+          def deserialize_arguments(job)
+            ::ActiveJob::Arguments.deserialize(job[:args]) rescue job[:args]
```

Comment:
Don't we want to only rescue `ActiveJob::DeserializationError`?

Reply:
Good catch, yes we do.